packages/lib/htmlUtils.ts

```diff
@@ -30,6 +30,17 @@ const selfClosingElements = [
 	'wbr',
 ];
 
+async function replaceAsync(str: string, regex: RegExp, asyncFn: Function) {
+	const promises: Promise<any>[] = [];
+	str.replace(regex, (match: string, ...args) => {
+		const promise = asyncFn(match, ...args);
+		promises.push(promise);
+		return null;
+	});
+	const data = await Promise.all(promises);
+	return str.replace(regex, () => data.shift());
+}
```

Comment:
Had to create this function to allow async callback in `replaceImageUrlsAsync` function.